packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -3243,6 +3243,39 @@ void main() {
             'snack_bar.goldenTest.floatingWithActionWithIcon.png'));
   });
 
+  testWidgets('SnackBar has tooltip for Close Button', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
```

Comment:
Per convention, could you add a comment here linking to the issue?

```suggestion
    // Regression test for https://github.com/flutter/flutter/issues/143793
    await tester.pumpWidget(MaterialApp(
```